go.mod

```diff
@@ -128,3 +128,5 @@ require (
 )
 
 go 1.20
+
+replace github.com/codeready-toolchain/api => github.com/metlos/api v0.0.0-20240708084953-3d53a2121f9f
```

Comment:
we should update the dependency as a separate PR - together with the changes in the code; otherwise, you merge api PR and then you would need to wait for another round of e2e tests. 
So ideally, in such PRs when we introduce only a new fields so it stays backward compatible with the logic in the code, the change would contain only the result of the `make generate` command.